CONTRIBUTING.md

```diff
@@ -1,7 +1,15 @@
 # Contributing to SMARTS
+🎉 First and foremost, thanks for taking the time to contribute! 🎉
 
-We encourage all forms of contributions to SMARTS, not limited to:
+The following is a set of concise guidelines for contributing to SMARTS and its packages. Feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+- Be respectful. 
+- Keep criticism strictly to code.
+- Ask a project lead before revealing information about the project to outside entities. 
```

Comment:
For open-source contributors, it's not obvious who the "project leads" are, nor what would constitute an "outside entity".